src/org/openlcb/swing/memconfig/MemConfigDescriptionPane.java

```diff
@@ -81,10 +85,14 @@ public void handleFailure(int code) {
                 public void handleConfigData(NodeID dest, int commands, int lengths, int highSpace, int lowSpace, String name) {
                     // fill window from values
                     commandLabel.setText("0x"+Utilities.toHexPair(commands>>8)+Utilities.toHexPair(commands));
+                    writesLabel.setText("0x"+Utilities.toHexPair(lengths));
 
                     highSpaceLabel.setText("0x"+Utilities.toHexPair(highSpace));
                     lowSpaceLabel.setText("0x"+Utilities.toHexPair(lowSpace));
 
+                    // sometimes highSpace doesn't include the pre-defined spaces,
+                    // so in the case we add them on and start with 0xFF
+                    if (highSpace >= 0xFA) highSpace = 0xFF;
```

Comment:
isn't this intended to be < 0xFA ?

Reply:
Certain nodes show 0xFA even though they have spaces up to 0xFF.  So this is meant as a Gentle Hack to include those upper spaces in the display.  Hence the = part of >= 0xFA.  The > part is an attempt at generality if some future node claims a top of 0xFB, etc.